articles/communication-services/quickstarts/call-automation/includes/callflow-for-customer-interactions-csharp.md

```diff
@@ -45,9 +45,22 @@ dotnet add <path-to-project> package Azure.Messaging.EventGrid
 
 In this quick-start, you'll use the new [Visual Studio Dev Tunnels](/connectors/custom-connectors/port-tunneling) feature to obtain a public domain name so that your local application is reachable by the Call Automation platform on the Internet. The public name is needed to receive the Event Grid `IncomingCall` event and Call Automation events using webhooks.
 
-If you haven't already configured your workstation, be sure to follow the steps in [this guide](/connectors/custom-connectors/port-tunneling). Once configured, your workstation will acquire a public domain name automatically allowing us to use the environment variable `["VS_TUNNEL_URL"]` as shown below.
+Open the application you created in the previous step in Visual Studio 2022. In the Solution Explorer, open the Properties – launchSettings.json and add the following lines in the https profile after the line  “"ASPNETCORE_ENVIRONMENT": "Development".
 
-Set up your Event Grid subscription to receive the `IncomingCall` event by reading [this guide](../../../concepts/call-automation/incoming-call-notification.md).
+```JSON
+"devTunnelEnabled": true,
+"devTunnelAccess":"Public"
+```
+
+Please consult [this guide](https://learn.microsoft.com/connectors/custom-connectors/port-tunneling) for more information. 
```

Comment:
Might be better to link to this blog instead since it is more general.  The connectors link has additional information not needed for call automation and it is missing a key step which the blog describes (enabling the preview feature in the first place in VS).